client/src/components/HomePage/Statistic/index.jsx

```diff
@@ -1,12 +1,39 @@
 import React, { Component } from 'react';
 import Fade from 'react-reveal';
+import axios from 'axios';
 import './style.css';
 import satistic from './satistic.jpeg';
 
 class Statistic extends Component {
-  state = {};
+  state = {
+    values: [],
+  };
+
+  componentWillMount() {
+    axios
+      .get('/api/v1/statistic')
+      .then((data) => {
+        const results = data.data;
+        this.setState({
+          values: results,
+        });
+      })
+      .catch((error) => {
+        const { data: message, status } = error.response;
+        if (status === 500) {
+          this.setState({ message });
+        }
```

Comment:
@ali-7 I think if there was an error and the status wasn't 500, you could still show a generic error message here.  That way, if heroku was down or something and the endpoint responded with (for example) `503 service unavailable` your app still does it's error displaying.